src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { Status } from '../../types/Status';
+import { Filter } from '../Filter';
+
+interface Props {
+  todos: Todo[];
+  filterBy: Status;
+  onAllclick: () => void;
+  onActiveClick: () => void;
+  onCompletedclick: () => void;
+}
+
+export const Footer: React.FC<Props> = React.memo(({
+  todos,
+  filterBy,
+  onAllclick,
+  onActiveClick,
+  onCompletedclick,
```

Comment:
The method name 'onCompletedclick' should follow camel case convention. Consider renaming to 'onCompletedClick' for consistency and readability.